client/src/modules/reports/generate/open_debtors/open_debtors.html

```diff
@@ -23,36 +23,101 @@ <h3 class="text-capitalize" translate>REPORT.OPEN_DEBTORS.TITLE</h3>
         <div class="panel-body">
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
 
-
-            <div class="form-group">
+            <div class="form-group" ng-class="{'has-error' : ConfigForm.order.$invalid && ConfigForm.$submitted}">
               <label class="control-label" translate>
-                FORM.PLACEHOLDERS.COLUMN
+                FORM.LABELS.ORDER_BY
               </label>
 
               <ui-select
```

Comment:
I've opted for `enabled-search="false"` on the ui-select.  It may not be perfect, but it allows the unicode characters to show through.

In the future, we could do any of these things:
 1. Write an CSS class to be applied to `:before` a `select > option` element that will put either a unicode caret up or caret down.
 2. Use text ("first", "last", "largest", "smallest", etc...)  These would need to be standardized across the entire application.